team_red/qa/qa_service.py

```diff
@@ -86,24 +88,22 @@ def query(self, question: QAQuestion) -> QAAnswer:
                         page=doc.metadata.get("page", 1),
                     )
                 )
-        if CONFIG.features.fact_checking:
+        if self._config.features.fact_checking.enabled is True:
             if not self._fact_checker_db:
-                self._fact_checker_db = self._setup_dbqa_fact_checking()
+                self._fact_checker_db = self._setup_dbqa_fact_checking(
+                    self._config.features.fact_checking.model.prompt
+                )
             response = self._fact_checker_db({"query": response["result"]})
             answer.answer = response["result"]
```

Comment:
I have the problem that when I activate factchecking, the answer on the website is now always empty. Does this have something to do with line 97? There the response of the model for the websitetextfield is overwritten with the response of the factcheck. But we want to display the answer and not the factcheck in this case. Just wondering why the response from the factcheck is empty...

Reply:
Okay. What should happen with the 'fact checking' response?

Reply:
According to your previous code 'fact checking' is just adding source documents to the answer. So for now, this is what  is done (again) in `query`.